documentation/en/default-lotus-config.toml

```
@@ -307,23 +292,64 @@
   # env var: LOTUS_EVENTS_MAXFILTERHEIGHTRANGE
   #MaxFilterHeightRange = 2880
 
-  # DatabasePath is the full path to a sqlite database that will be used to index actor events to
-  # support the historic filter APIs. If the database does not exist it will be created. The directory containing
-  # the database must already exist and be writeable. If a relative path is provided here, sqlite treats it as
-  # relative to the CWD (current working directory).
+
+[ChainIndexer]
+  # EnableIndexer controls whether the chain indexer is active.
+  # The chain indexer is responsible for indexing tipsets, messages, and events from the chain state.
+  # It is a crucial component for optimizing Lotus RPC response times.
+  # 
+  # Default: false (indexer is disabled)
+  # 
+  # Setting this to true will enable the indexer, which will significantly improve RPC performance.
+  # It is strongly recommended to keep this set to true if you are an RPC provider.
   #
-  # type: string
-  # env var: LOTUS_EVENTS_DATABASEPATH
-  #DatabasePath = ""
+  # type: bool
+  # env var: LOTUS_CHAININDEXER_ENABLEINDEXER
+  #EnableIndexer = false
 
+  # GCRetentionEpochs specifies the number of epochs for which data is retained in the Indexer.
+  # The garbage collection (GC) process removes data older than this retention period.
+  # Setting this to 0 disables GC, preserving all historical data indefinitely.
+  # 
+  # If set, the minimum value must be greater than builtin.EpochsInDay (i.e. "2880" epochs for mainnet).
```

```suggestion
  # If set, the minimum value must be greater than a day's worth (i.e. "2880" epochs for mainnet).
```
leaky implementation detail